main.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'app'
+
+def show_options
+  puts ''
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+def main
+  app = App.new
+
```

Comment:
- [ ] As per the project requirements, please define the entry point in main.rb. This should be a method called "main" that will be invoked at the end of the file. The main method should perform the following tasks:
  - Display a list of options for the user to choose from.
  - Allow the user to select an option.
  - If necessary, prompt the user for any required parameters.
  - Provide a way for the user to exit the application.
  
  However, please note that when I attempt to start your application using "ruby main.rb", I get this error message below :point_down:  . Please ensure that this issue is resolved and that the app functions correctly according to the requirements.
    
    ![image](https://github.com/MMGGYY66/oop-school-lib/assets/92303891/f3701dfb-838d-423a-b230-bce59a8d2605)

Reply:
it is a propblem in your computer. it is working with me fine and properly. try fix your computer